spec/linter-eslint-spec.js

```diff
@@ -49,14 +49,12 @@ function copyFileToDir(fileToCopyPath, destinationDir) {
  * @return {string}                 Full path of the file in copy destination
  */
 // eslint-disable-next-line import/prefer-default-export
-export function copyFileToTempDir(fileToCopyPath) {
-  return new Promise(async (resolve) => {
-    const tempFixtureDir = fs.mkdtempSync(tmpdir() + path.sep)
-    resolve(await copyFileToDir(fileToCopyPath, tempFixtureDir))
-  })
+export async function copyFileToTempDir(fileToCopyPath) {
+  const tempFixtureDir = fs.mkdtempSync(tmpdir() + path.sep)
+  return copyFileToDir(fileToCopyPath, tempFixtureDir)
 }
 
-async function getNotification(expectedMessage) {
+function getNotification(expectedMessage) {
```

Comment:
Following the rest of the discussion, take out the removal of `async` here 😉.

Reply:
Done. This is rebased and ready to go as soon as the CI checks are done.

Reply:
You got the second one, but seem to have left this removal in 😛.

Reply:
done